src/native/libs/build-local.sh

```diff
@@ -4,24 +4,25 @@
 # The .NET Foundation licenses this file to you under the MIT license.
 #
 
-# This script is used only for building libSystem.Globalization.Native.a
+# This script is used only for building native libraries
 # in the end-user's computer for NativeAOT purposes (static linking).
 # This file is not used during the dotnet runtime build.
 
 # Currently, only Linux is supported
 
-SHIM_SOURCE_DIR="$1"/native/src
-INTERMEDIATE_OUTPUT_PATH="$2"
+SHIM_SOURCE_DIR=$1/native/src
+INTERMEDIATE_OUTPUT_PATH=$2
+TARGET_LIBRARY=$3
```

Comment:
```suggestion
SHIM_SOURCE_DIR="$1"/native/src
INTERMEDIATE_OUTPUT_PATH="$2"
TARGET_LIBRARY="$3"
```

For paths with spaces (and to make `shellcheck(1)` happy 🙂).